pkg/operator/apiserver/audit/audit_policies.go

```diff
@@ -83,11 +83,26 @@ func GetAuditPolicy(audit configv1.Audit) (*auditv1.Policy, error) {
 	p := basePolicy.DeepCopy()
 	p.Name = string(audit.Profile)
 
-	extraRules, ok := profileRules[audit.Profile]
+	for _, cr := range audit.CustomRules {
+		profileRules, ok := profileRules[audit.Profile]
+		if !ok {
+			return nil, fmt.Errorf("unknown audit profile %q", audit.Profile)
+		}
+
+		groupRules := make([]auditv1.PolicyRule, len(profileRules))
+		for i, r := range profileRules {
+			r.DeepCopyInto(&groupRules[i])
+			groupRules[i].UserGroups = []string{cr.Group}
```

Comment:
what if in the future we decide to have a rule that already sets `UserGroups` ?
we could append instead of overwriting

Reply:
ah, there's a unit test for that `TestNoUserGroups`

Reply:
See my rant about missing conjunction in Slack channel. These profiles for group can never have pre-defined groups. We made a mistake in API design long ago in upstream.